app/javascript/components/App.js

```diff
@@ -13,6 +13,60 @@ import MetricsEdit from "./pages/MetricsEdit";
 import MockMetrics from "./MockMetrics";
 
 const App = (props) => {
+  const [metrics, setMetrics] = useState([]);
+  useEffect(() => {
+    readMetrics();
+  }, []);
+
+  const readMetrics = () => {
+    fetch("/metrics")
+      .then((response) => response.json())
+      .then((payload) => {
+        setMetrics(payload);
+      })
+      .catch((error) => console.log(error));
+  };
+
+  const createMetric = (metric) => {
+    console.log(metric)
+    fetch("http://localhost:3000/metrics", {
```

Comment:
For read you do not have localhost:3000 as part of your url which is great since this is a monolithic application.  Please update the rest of your fetch urls to be consistent.